int8 enable_adas

int8 enable_acc

int8 enable_aeb

int8 enable_lka

float32 tar_v_cc
